css/index.css

```diff
@@ -71,6 +162,31 @@ p {
     margin: 0 auto;
 }
 
+.index-page-header {
+    display: flex;
+    justify-content: flex-start space-around;
```

Comment:
This style isn't being applied right now because you would only be able to use flex-start or space-around. The header still looks good though! 👍